.github/workflows/go_lint.yml

```diff
@@ -1,6 +1,8 @@
 name: Go Linting
 
-on: [pull_request]
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
```

Comment:
[Default event types](https://docs.github.com/en/actions/writing-workflows/choosing-when-your-workflow-runs/events-that-trigger-workflows#pull_request) include `opened, synchronize, reoponed`